### ISSUE TITLE
cannot run tests.py or basic_mer_tests.py

### ISSUE THREAD

**Issue by pebbie** [RyanBalfanz/PyPDS#14]
in the Examples part of the README.rst, it stated to run test but it seems that it always failed because there's no test_data directory/files given in the distribution.

**Comment by RyanBalfanz** [comment]
Yeah, the files I was using we're pretty big. It didn't make sense to include them in the repository directly. There used to be a downloads section on Github, but I can't seem to find it. There were a bunch of PDS files there.

**Comment by godber** [comment]
I have made a tarball containing the files included in the tests.  You can access it here:

http://uberhip.com/_static/pypds-test-data.tgz

Distributing these is not that big of a deal, but if the test suite grows, as it should, perhaps we should have a utility that grabs the products from their public repos and builds the test directory.
